Resources/config/form-type.xml

```diff
@@ -9,5 +9,13 @@
             <tag name="form.type" alias="cmf_core_checkbox_url_label" />
             <argument type="service" id="router" />
         </service>
+
+        <service id="cmf_core.form.extension.publishable" abstract="true" class="Symfony\Cmf\Bundle\CoreBundle\Form\Extension\PublishableExtension">
+            <!-- <argument /> Extended type fqcn -->
```

Comment:
The `FormTypeExtension` cannot work for an array of types, however, we can use some configuration and create services using `cmf_core.form.extension.publishable` as parent service.

@wourterj the argument in the abstract service is an example. I don't think it's good to set an invalid argument and forcing to override it.